Add De Lijn (Flemish Public Transport) component

bollewolle opened the pull request:
## Breaking Change:

none

## Description:
Add [De Lijn](https://www.delijn.be) (the Flemish Public Transport agency) support. The use of the API's of De Lijn does require a subscription key that needs to be added to the configuration. You can register for a free developer account at https://data.delijn.be/ where you can find the subscription key.

Works best with the following custom lovelace card: https://github.com/bollewolle/delijn-card

**Related issue (if applicable):** fixes #<home-assistant issue number goes here>

**Pull request with documentation for [home-assistant.io](https://github.com/home-assistant/home-assistant.io) (if applicable):** https://github.com/home-assistant/home-assistant.io/pull/9507

## Example entry for `configuration.yaml`:
```yaml
sensor:
  # De Lijn public transport
  - platform: delijn
    sub_key: 'abcdefg'
    nextpassage:
    - stop_id: '200018'
      max_passages: 5
    - stop_id: '201169'
      max_passages: 20
```
Replace the abcdefg with the API subscription key of De Lijn.
Stop Id's are 6 digit number that can be found at each physical stop of De Lijn in Flanders, but can also be found on their site here: https://www.delijn.be/en/haltes/

## Checklist:
  - [x] The code change is tested and works locally.
  - [x] Local tests pass with `tox`. **Your PR cannot be merged unless tests pass**
  - [x] There is no commented out code in this PR.
  - [x] I have followed the [development checklist][dev-checklist]

If user exposed functionality or configuration variables are added/changed:
  - [x] Documentation added/updated in [home-assistant.io](https://github.com/home-assistant/home-assistant.io)

If the code communicates with devices, web services, or third-party tools:
  - [x] [_The manifest file_][manifest-docs] has all fields filled out correctly. Update and include derived files by running `python3 -m script.hassfest`.
  - [x] New or updated dependencies have been added to `requirements_all.txt` by running `python3 -m script.gen_requirements_all`.
  - [ ] Untested files have been added to `.coveragerc`.

If the code does not interact with devices:
  - [ ] Tests have been added to verify that the new code works.

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[manifest-docs]: https://developers.home-assistant.io/docs/en/creating_integration_manifest.html

pvizeli committed:
Fix Hassio-version for Azure Pipelines (#23895)

balloob committed:
Update Honeywell warning (#23913)

elupus committed:
Switch media player to SWITCH type (#23914)

MEDIA device type is being rejected by google now.

karlkar committed:
Fix problem with cameras that don't support time (#23924)

Some onvif cameras don't support Date management. In that case None is returned and script crashes when trying to obtain date

balloob committed:
Bumped version to 0.93.1

balloob committed:
Merge pull request #23932 from home-assistant/rc

0.93.1

pvizeli committed:
Fix auto version update Hass.io (#23935)

ludeeus committed:
Bump pytraccar (#23939)

aerialls committed:
Setup integration dependencies before loading it (#23957)

Danielhiversen committed:
upgrade broadlink library (#23966)

cyrosy committed:
Make Discord payload data key not required (#23964)

ludeeus committed:
Fixes issue with multiple alerts (#23945)

* Fixes issue with multiple alerts

* Adds missing new line

* Remove whitespace

balloob committed:
Bumped version to 0.93.2

balloob committed:
Merge pull request #24047 from home-assistant/rc

0.93.2

bollewolle committed:
Add De Lijn (Flemish Public Transport) component

bollewolle committed:
Add De Lijn (Flemish Public Transport) component

homeassistant commented:
Hello @bollewolle,

When attempting to inspect the commits of your pull request for CLA signature status among all authors we encountered commit(s) which were not linked to a GitHub account, thus not allowing us to determine their status(es).

The commits that are missing a linked GitHub account are the following:

- [`5613be7effacd1ae40b318f177b3257ef534fc84`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;home-assistant&#x2F;pull&#x2F;24067&#x2F;commits&#x2F;5613be7effacd1ae40b318f177b3257ef534fc84) - This commit has something that looks like an email address (bram@goolaerts.be). Maybe try linking that to GitHub?.

Unfortunately, **we are unable to accept this pull request until this situation is corrected.**

Here are your options:

1. If you had an email address set for the commit that simply wasn't linked to your GitHub account you can link that email now and it will retroactively apply to your commits. The simplest way to do this is to click the link to one of the above commits and look for a blue question mark in a blue circle in the top left. Hovering over that bubble will show you what email address you used. Clicking on that button will take you to your email address settings on GitHub. Just add the email address on that page and you're all set. GitHub has more information about this option [in their help center](https://help.github.com/articles/setting-your-email-in-git/#commits-on-github-arent-linking-to-my-account).

2. If you didn't use an email address at all, it was an invalid email, or it's one you can't link to your GitHub, you will need to change the authorship information of the commit and your global Git settings so this doesn't happen again going forward. GitHub provides some great instructions on how to change your authorship information [in their help center](https://help.github.com/articles/setting-your-email-in-git/).
   * If you only made a single commit you should be able to run
      ```
      git commit --amend --author="Author Name <email@address.com>"
      ```
      (substituting Author Name and email@address.com for your actual information) to set the authorship information.
   * If you made more than one commit and the commit with the missing authorship information is not the most recent one you have two options:
       1. You can re-create all commits missing authorship information. This is going to be the easiest solution for developers that aren't extremely confident in their Git and command line skills.
       2. You can use [this script](https://help.github.com/articles/changing-author-info/) that GitHub provides to rewrite history. **Please note:** this should be used only if you are very confident in your abilities and understand its impacts.
   * Whichever method you choose, I will come by to re-check the pull request once you push the fixes to this branch.

We apologize for this inconvenience, especially since it usually bites new contributors to Home Assistant. We hope you understand the need for us to protect ourselves and the great community we all have built legally. The best thing to come out of this is that you only need to fix this once and it benefits the entire Home Assistant and GitHub community.

Thanks, I look forward to checking this PR again soon! :heart:

balloob committed:
Updated frontend to 20190523.0

balloob committed:
Update translations

fredrike committed:
Daikin airbase beta fixes (#24050)

* values are strings not integers

* pydaikin version bump

JeffLIrion committed:
Better logging of method used for ADB connection (#24037)

balloob committed:
Fix zeroconf sorting (#24068)

bollewolle committed:
Incorrect accidental code

Small correction of an accidental code change

bollewolle committed:
Incorrect accidental code

Small correction of an accidental code change

homeassistant commented:
Hello @bollewolle,

When attempting to inspect the commits of your pull request for CLA signature status among all authors we encountered commit(s) which were not linked to a GitHub account, thus not allowing us to determine their status(es).

The commits that are missing a linked GitHub account are the following:

- [`5613be7effacd1ae40b318f177b3257ef534fc84`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;home-assistant&#x2F;pull&#x2F;24067&#x2F;commits&#x2F;5613be7effacd1ae40b318f177b3257ef534fc84) - This commit has something that looks like an email address (bram@goolaerts.be). Maybe try linking that to GitHub?.

Unfortunately, **we are unable to accept this pull request until this situation is corrected.**

Here are your options:

1. If you had an email address set for the commit that simply wasn't linked to your GitHub account you can link that email now and it will retroactively apply to your commits. The simplest way to do this is to click the link to one of the above commits and look for a blue question mark in a blue circle in the top left. Hovering over that bubble will show you what email address you used. Clicking on that button will take you to your email address settings on GitHub. Just add the email address on that page and you're all set. GitHub has more information about this option [in their help center](https://help.github.com/articles/setting-your-email-in-git/#commits-on-github-arent-linking-to-my-account).

2. If you didn't use an email address at all, it was an invalid email, or it's one you can't link to your GitHub, you will need to change the authorship information of the commit and your global Git settings so this doesn't happen again going forward. GitHub provides some great instructions on how to change your authorship information [in their help center](https://help.github.com/articles/setting-your-email-in-git/).
   * If you only made a single commit you should be able to run
      ```
      git commit --amend --author="Author Name <email@address.com>"
      ```
      (substituting Author Name and email@address.com for your actual information) to set the authorship information.
   * If you made more than one commit and the commit with the missing authorship information is not the most recent one you have two options:
       1. You can re-create all commits missing authorship information. This is going to be the easiest solution for developers that aren't extremely confident in their Git and command line skills.
       2. You can use [this script](https://help.github.com/articles/changing-author-info/) that GitHub provides to rewrite history. **Please note:** this should be used only if you are very confident in your abilities and understand its impacts.
   * Whichever method you choose, I will come by to re-check the pull request once you push the fixes to this branch.

We apologize for this inconvenience, especially since it usually bites new contributors to Home Assistant. We hope you understand the need for us to protect ourselves and the great community we all have built legally. The best thing to come out of this is that you only need to fix this once and it benefits the entire Home Assistant and GitHub community.

Thanks, I look forward to checking this PR again soon! :heart:

bollewolle commented:
Email address of the commit has been added to Github and a small code error has been changed.

Danielhiversen committed:
Rfxtrx, add data types (#24066)

* Rfxtrx, add data types

* fix style

dreed47 committed:
Update the name of Zestimate sensors (#23770)

* Zestimate: fix for issue #23757
Changed name property to return Zestimate
and the property address.  This will make it easier
distinguish multiple Zestimate sensor entities
in the UI.

Also removed MIN_TIME_BETWEEN_UPDATES in
favor of SCAN_INTERVAL per suggestion from
amelchio#9580 on Discord

* Zestimate fix for issue #23757

Changed name property to return Zestimate
and the property address.  This will make it easier
distinguish multiple Zestimate sensor entities
in the UI.

* Changed name property to return Zestimate
and the property address.  This will make it easier
distinguish multiple Zestimate sensor entities
in the UI.

* moved code fix to the correct function

* removed code change from unique_id function

p0l0 committed:
Added possibility to define the data type of Homematic (#24078)

* Homematic: Added possibility to define the data type for set_device_value

* Fixed coding style

* Fixed variable name

JeffLIrion committed:
Add 'adb_response' attribute to Android TV / Fire TV (#23960)

* Add 'adb_response' attribute to Android TV / Fire TV

* Use None instead of empty string for empty ADB responses

* Initialize self._adb_response as None, not empty string

* Update the state after sending an ADB command

This ensures that the `'adb_response'` attribute contains the response to the latest command

elupus committed:
Adjust logging (#24082)

* Make sure we log full path to debug log

* Make sure we log the exception to debug log

Commit:
Fix Hue bridge timeout (#24084)

* Change timeout from 5 seconds to 10 seconds
Underpowered platforms timeout during configuration/discovery of a Hue bridge on a new install. Increasing this timeout fixes this.

jardiamj committed:
MCP23017 (#23127)

* Added support for MCP23017 I2C GPIO extender.

* Updated .coveragerc to exclude mcp23017 component from tests.

* Generated CODEOWNERS for mcp23017 usign script.

* Removed .svn folder that had been accidentally uploaded.

* Added link to www.home-assistant.io docs in manifest.json

* Fixed logic error in switch platform.

* Cleaned up code and removed unnecessary should_poll() function.

* Limited the options for pull mode to UP and DOWN

* Fixed line too long in binary sensor.

* Fixed line too long on switch.py

* Changed to setup_platform.

* Reorder constants

Danielhiversen committed:
typo for ambiclimate (#24083)

bollewolle commented:
Not sure if I've overlooked anything but is there still an issue I need to solve for this PR?

elupus committed:
Remove device tracker unnecessary separate except clause (#24081)

Handle exception where it can be thrown.

alengwenus committed:
Refactoring of LCN component (#23824)

* Moved helper functions to const.py

* Removed pypck attribute from LcnDevice

* Bump to pypck==0.6.0

* Added myself as a codeowner

* Moved helper functions to helpers.py

simse committed:
Update code owner for Xiaomi TV (#24102)

* Update code owner

* Update CODEOWNERS

fronzbot committed:
Fix broken blink motion detection (#24097)

techfreek committed:
Issue #23514 - fix invalid hue response (#23909)

Based on the discoveries in issue #23514, the periodic lack of response from emulated hue was due to an invalid value (null) being returned.

balloob committed:
Config entry device tracker (#24040)

* Move zone helpers to zone root

* Add config entry support to device tracker

* Convert Geofency

* Convert GPSLogger

* Track unsub per entry

* Convert locative

* Migrate OwnTracks

* Lint

* location -> latitude, longitude props

* Lint

* lint

* Fix test

squishykid committed:
Solax Inverter Sensor Component (#22579)

* Solax inverter direct API

* Linter compliance

* lint++

* move api communication to external lib

* lint++

* requirements

* Revert "requirements"

This reverts commit 82a6c0c095ae5d454a1a16f72a77941626fa5ea2.

* potentially?

* Addressing review comments

* Also update CODEOWNERS

* Only update sensor state if data has changed

cgtobi committed:
Bump pyatmo version to 1.12 (#24088)

jardiamj committed:
Set assumed_state property to True. (#24118)

jjlawren committed:
Remove custom entity_id naming (#24072)

* Remove custom entity_id naming

* Set entity_ids with 'plex'

* Set name instead of entity_id

* Lint

* Use a name template

andrewsayre committed:
Move imports to top (#24108)

OttoWinter committed:
Use name in ESPHome discovery title (#24100)

* Use name in ESPHome discovery title

* Add test

* Lint

jgriff2 committed:
Add Remote RPi Component (#23518)

* Add Remote RPi Component

* Add Remote RPi Component

* fix imports

* Added support for setup as switch and binary_sensor

* remove pylint error handling

* Changed to domain config

* Changed to domain config

* Changed to domain config

* Changed to domain config

* Update __init__.py

* Update manifest.json

* Update requirements_all.txt

* Update switch.py

* Update binary_sensor.py

* Changed to domain config

eavanvalkenburg committed:
Azure Event Hub history component (#23878)

* v1 of Azure Event Hub History component

* updates to EH

* small fix

* small updates and changed requirements_all

* new version of Event Hub component

* redid config to just ask names

* small edit

* latest version of EH component

* updated codeowners

* codeowner fix

* typo in domain

* updates based on reviews.

* using built-in jsonencoder for DT

* delete unused import

zxdavb committed:
geniushub: fix sensor battery level, and bump client (#24123)

* Initial commit

* bump client

balloob committed:
Use importlib metadata to check installed packages (#24114)

* Use importlib metadata

* Fix script

* Remove unused import

* Update requirements"

amelchio committed:
Avoid useless Sonos state updates (#24135)

balloob committed:
Add SSDP integration (#24090)

* Add SSDP integration

* Fix tests

* Sort all the things

* Add netdisco to test requirements

bramkragten committed:
Lovelace: Fire event on save (#24104)

* Lovelace: Fire event on save

* Add event to whitelist

jjlawren committed:
Use central polling to update entities (#24059)

* Use central polling to update entities

* Fix for line length

* Remove unnecessary import

* Use interval

* Trigger entity refreshes after commands

* Lint

Kane610 committed:
Library refactorization of deCONZ (#23725)

* Improved sensors

* Lib update signalling

* Replace reason with changed

* Move imports to top of file

* Add support for secondary temperature reported by some Xiaomi devices

* Bump dependency to v59

aerialls committed:
Retrieve wire and wireless devices with the SRM device tracker (#24117)

maheus committed:
Add station name for creating the unique_id in netatmo platform (#24141)

jesserizzo committed:
bump dependency envoy_reader to 0.4 (#24145)

* bump envoy_reader version to 0.4

* bump dependency envoy_reader to 0.4

TheLastProject committed:
Don't crash on first EAN without installations (#24137)

* Don't crash on first EAN without installations

* Remove duplicated values

* Switch from Exception to persistent notification

* Make pylint happy

Commit:
pubnubsub-handler to 1.0.5 (#24154)

Commit:
Pubnub to 1.0.6 (#24159)

Adminiuga committed:
Debug log when polling ZHA light. (#24167)

chmielowiec committed:
Upgrade huawei-lte-api to 1.2.0 (#24165)

robbiet480 committed:
Use device name for device_tracker entry (#24155)

OttoWinter committed:
Use global imports for ESPHome (#24158)

* Use global import for ESPHome

* Add aioesphomeapi to test requirements

aioesphomeapi is also shipped as a pure-python wheel, so this should not impact test install time

MTrab committed:
Add Repetier-Server Component (#21658)

* Bumped to version 2.0

* Updated requirements

* Updated requirements and coveragerc

* Removed long lines, changes to coveragerc and requirements

* Fixed under-indented lines

* Fixed invalid syntax

* Updated .coveragerc to include repetier/__init__.py and sensor.py

* Module update

* Rebased to latest dev

* Blank lines fix

* Add missing manifest.json

* Update requirements

* Bumped component to new API module style

* Removed whitespaces and line feeds

* Added missing newline

* Added missing heated chamber sensor

* Fixed wrong indentation

* Various fixes

* Various build fixes

* Clean up

* Load platform only once

* Sort imports

* Add printer api

* Clean up

* Build out sensor classes

* Clarify temperature sensor variable names

* Move constants

* Clean up name

* Run script/gen_requirements_all.py

* Working code, missing auto add of new sensors

* Updated code to return proper device_class and timestamp

* Removed unnessecary code

* Renamed elapsed and remaining sensors

* Dynamically adding sensors as they become available

* Rebased .coveragerc due to conflicts

* Code changes and cleanup

* Removed whitespace and code simplification

balloob committed:
Cloud: Websocket API to manage Google assistant entity config (#24153)

* Extract exposed devices function

* Add might_2fa info to trait

* Do not filter with should_expose in Google helper func

* Cloud: allow setting if Google entity is exposed

* Allow disabling 2FA via config

* Cloud: allow disabling 2FA

* Lint

* More changes

* Fix typing

robbiet480 committed:
Fix calling notify.notify with mobile_app targets in play. Fixes #24064 (#24156)

balloob committed:
Updated frontend to 20190529.0

balloob committed:
update translations

amelchio committed:
Remove unused Sonos turn on/off methods (#24174)

balloob committed:
Reinstate passing loop to DSMR (#24127)

* Reinstate passing loop

* Also pass loop into other part

Kane610 committed:
Trådfri component to use new zeroconf discovery (#24041)

* Move tradfri component to use new zeroconf discovery

* Will this work?

* Remove prints

* Correct order in generated zeroconf

* Update test_init.py

* Update test_init.py

* Update test_init.py

* Update test_init.py

Kane610 committed:
Move Homekit controller component to user zeroconf discovery (#24042)

Kane610 committed:
Revert Zeroconf back to previously used library (#24139)

* Revert back to previously used library

* Fix test

* Remove unused import

* Fix import

* Update __init__.py

* Update __init__.py

* Fix test after rebase

balloob committed:
Deprecate Python 3.5.3 (#24177)

Kane610 committed:
Keep integrations in discovery (#24179)

* Keep integrations that have been migrated to new discovery methods to avoid breaking changes

* Additional migrated services

amelchio committed:
Avoid slow updates with unavailable Sonos devices (#24180)

balloob committed:
Merge remote-tracking branch 'origin/master' into dev

pvizeli committed:
Support Hass.io wheels / docker env (#24175)

* Support Hass.io wheels / docker env

* address comments

* fix lint

balloob committed:
Remove discovery from initial config (#24183)

Jc2k committed:
Fix duplicated discovered homekit devices (#24178)

alengwenus committed:
Add service calls for LCN component (#24105)

amelchio committed:
Update pysonos to 0.0.14 (#24185)

Danielhiversen committed:
upgrade broadlink library, Use cryptography instead of pycryptodome (#24186)

Danielhiversen committed:
danielhiversen as codeowner for yr (#24189)

balloob committed:
No longer allow invalid slugs or extra keys (#24176)

* No longer allow slugs

* Lint

* Remove HASchema

* Lint

* Fix tests

balloob committed:
Dynamic panels (#24184)

* Don't require all panel urls to be registered

* Allow removing panels, fire event when panels updated

pvizeli committed:
Update azure-pipelines.yml for check version (#24194)

balloob committed:
Updated frontend to 20190530.0

Jc2k committed:
homekit_controller no longer logs with transient network errors causing crypto failures as it will auto recover (#24193)

Danielhiversen committed:
Update Tibber library (#24192)

frenck committed:
:pencil2: Corrects incorrect command in hassfest (#24188)

pvizeli committed:
Update azure-pipelines.yml for Azure Pipelines

OttoWinter committed:
Fix ESPHome discovered when already exists (#24187)

* Fix ESPHome discovered when already exists

* Update .coveragerc

balloob committed:
Update hass-nabucasa (#24197)

balloob committed:
Allow discovery flows to be discovered via zeroconf/ssdp (#24199)

balloob committed:
Improve error handling (#24204)

vangorra committed:
Fixing tplink issues with offline devices during setup (#23668)

* Fixing tplink issues with offline devices during setup.

* Fixing circleci errors.

* Adding code to defer the creation of entities that are not online.

* Addressing code review changes and cleaning up a little.

* Fixing tests and static analysis.

* Adding test to satisfy coverage requirements.

* Resolving merge conflicts.

* Fixing issue where lights don't appear in the integration page.

* Using pyHS100 properties for most sysinfo.
Addressing some PR feedback.

* Addressing some PR feedback.

* Better testing async_add_entities_retry
Testing for static dimmers.
Making greater use of conf constants.

* Fixing all static analysis issues.

* Adding non-blocking call for getting discovering devices.

* Address PR feedback

therve committed:
Bump oauthlib version (#24111)

* Bump oauthlib version

* Bump fitbit instead

* Update requirements

pvizeli committed:
Update azure-pipelines.yml for Azure Pipelines

OttoWinter committed:
Fix ESPHome config flow with invalid config entry (#24213)

pvizeli committed:
Don't follow redirect on ingress itself (#24218)

* Don't follow redirect on ingress itself

* Fix comment

pvizeli committed:
Update azure-pipelines.yml for Azure Pipelines

pvizeli committed:
Update azure-pipelines.yml for Azure Pipelines

balloob committed:
Instantiate lock inside event loop (#24203)

balloob committed:
Use resource for index routing. (#24223)

Kane610 committed:
Axis - Handle Vapix error messages (#24215)

balloob committed:
Add manifest support for homekit discovery (#24225)

* Add manifest support for homekit discovery

* Add a space after model check

* Update comment

pnbruckner committed:
Add control of Amcrest indicator light (#23986)

Enable feature by default but allow it to be disabled by "control_light: false" in config.

Get brand from camera instead of assuming Amcrest (since this works with other cameras, too.)

Retrieve RTSP URL in update method instead of in stream_source property and in handle_async_mjpeg_stream method.

Move amcrest imports from methods to global.

pvizeli committed:
Update azure-pipelines.yml for Azure Pipelines

Adminiuga committed:
ZHA requirements version bump. (#24228)

* ZHA requirements version bump.
* zha-quirks version bump.

Kane610 committed:
Don't allow more than one config flow per discovered Axis device (#24230)

soldag committed:
Bump pychromecast (#24234)

balloob committed:
Log HomeKit model (#24229)

balloob committed:
Add GPSLogger device_info and unique_id (#24231)

balloob committed:
GeoFency unique ID and device info (#24232)

rytilahti committed:
add a deprecation warning for tplink device_tracker (#24236)

* add a deprecation warning for tplink device_tracker

* reword the warning a bit

balloob committed:
Mobile app to use device tracker config entry (#24238)

* Mobile app to use device tracker config entry

* Lint

* Re-use device_info

* Lint

balloob committed:
Update home zone when core config updated (#24237)

* Update home zone when core config updated

* Lint

balloob committed:
Do not use the cache dir for PIP installs (#24233)

danielsjf committed:
Round Awair sensor values (#24093)

* Round sensor values

* Add code owner

* Update code owners

* Fix tests

pvizeli committed:
Update azure-pipelines.yml for Azure Pipelines

kbickar committed:
Sense update (#24220)

* Changed updates to be done by component and updates dispatched

* syntax updates

* Added code owner

* Removed whitespace

* Updated CODEOWNERS

* Added subscription undoer

Emilv2 committed:
fix gitlab_ci sad icon (#24241)

austinmroczek committed:
Add details to triggered state for total connect alarms (#24106)

* Bump skybellpy to 0.4.0

* Bump skybellpy to 0.4.0 in requirements_all.txt

* Added extra states for STATE_ALARM_TRIGGERED to allow users to know if
it is a burglar or fire or carbon monoxide so automations can take
appropriate actions.  Updated TotalConnect component to handle these new
states.

* Fix const import

* Fix const import

* Fix const imports

* Bump total-connect-client to 0.26.

* Catch details of alarm trigger in state attributes.

Also bumps total_connect_client to 0.27.

* Change state_attributes() to device_state_attributes()

cereal2nd committed:
Bump python-velbus version for velbus component (#24226)

* Bump python-velbus version

* Bump python velbus version

pvizeli committed:
Update azure-pipelines.yml for Azure Pipelines

balloob committed:
Updated frontend to 20190601.0

frenck committed:
Adds AdGuard Home integration (#24219)

* Adds AdGuard Home integration

* :shirt: Addresses linting warnings

* :ambulance: Fixes typehint in async_setup_entry

* :shirt: Take advantage of Python's coalescing operators

* :shirt: Use adguard instance from outer scope directly in service calls

* :shirt: Use more sensible scan_interval default for sensors

* :shirt: Adds specific files to .coveragerc

* :umbrella: Added tests and small changes to improve coverage

* :hammer: Import adguardhome dependencies at the top

* :ambulance: Converted service handlers to be async

* :fire: Removed init step from config flow

Emilv2 committed:
Fix typo in integration component (#24250)

pvizeli committed:
Update azure-pipelines.yml for Azure Pipelines

bollewolle committed:
updated coeragerc to indicate lack of tests

bollewolle committed:
Merge branch 'delijn' of https://github.com/bollewolle/home-assistant into delijn
